pkg/standalone/uninstall.go

```diff
@@ -45,9 +47,13 @@ func Uninstall(uninstallAll bool, dockerNetwork string) error {
 		fmt.Println("WARNING: could not delete run data file")
 	}
 
-	if len(failedContainers) == 0 {
+	if len(errs) == 0 {
 		return nil
 	}
 
-	return fmt.Errorf("could not delete (%s)", strings.Join(failedContainers, ","))
+	err = errors.New("uninstall failed")
+	for _, e := range errs {
+		err = fmt.Errorf("%w; %s", err, e)
```

Comment:
Can we please format the error this way:

```
err = errors.New("uninstall failed:")
for _, e := range errs {
	err = fmt.Errorf("%w \n %s", err, e)
```

Reply:
@pruthvidhodda Makes sense 👍 Thanks 🙂